### ISSUE TITLE
Navbar is not responsive

### ISSUE THREAD

**Issue by Axxi3** [Axxi3/Book-Exchange-System#1]
I want to make the navbar responsive

**Comment by Sky-De** [comment]
@Axxi3 hi, assign this to me, i would like to fix it,thanks

**Comment by Axxi3** [comment]
@Sky-De yeah sure man if you want you can do it

**Comment by Axxi3** [comment]
@Sky-De if you need any help feel free to ping me

**Comment by Axxi3** [comment]
Issue was fixed by @Sky-De

**Comment by Axxi3** [comment]
hey @Sky-De i checked the nav bar its not closing when i clicked on it can you fix that??

**Comment by Sky-De** [comment]
@Axxi3 it was working, i will check and inform you

**Comment by Axxi3** [comment]
@Sky-De i tried cloning it on my local machine and when i checked it on my phone the menu was not closing even though i clicked on it and also can you move the search bar with the navitems so that it is only visible when clicked on hamburger icon

**Comment by Sky-De** [comment]
@Axxi3 hi again, i just checked it out, the branch which i used for PR is working fine in both mobile and desktop,
but after pulling new commits into main branch it seems you did not merge my changes or in other commits may change the functionality and styles
please check other commits and found bug
if you could not find it let me know to make another pull request for this issue again
thanks

**Comment by Axxi3** [comment]
@Sky-De can you create another pull request

**Comment by Sky-De** [comment]
@Axxi3 ok, i will do it , but conseider do not merge any other PR before mine , i will send PR today, ok?

**Comment by Sky-De** [comment]
@Axxi3 i think this issue is fixed, please close this
thanks
